_posts/2017/06/2017-06-17-merger.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+authors: ["Karen Cranston", "Rayna Harris", "Kate Hertweck", "Hilmar Lapp"]
+title: "Timeline for the the Data Carpentry & Software Carpentry Reorganization"
+date: 2017-06-17
+time: "16:00"
+category: ["Steering Committee", "Community", "Data Carpentry"]
+---
+
+On Jun 8 & 9, we met in-person to continue discussing how to proceed with the merger of the Software Carpentry and Data Carpentry organizations to best achieve our strategic goals and serve our communities. We are happy to report that is was a very productive meeting. 
+ 
+We spent most of the meeting discussing the structure of the future organizational leadership. We have drafted a number of motions that we will present to our respective boards for approval at our next board meetings. Among the motions, we will propose is that the combined Carpentries governing board structure be made up of both appointed and elected members. Additionally, we propose that it is in the best interest of the organization to seed the future governing board with bylaws, which will be enumerated, written, and approved in the coming months. 
+ 
+Below is a timeline for the merger process. Various steps in this timeline require approval by the two governing boards, and are dependent on results of previous steps, so this may change as we proceed. 
+ 
+| Month | Merger Process Milestones | 
+| --- | --- | 
+|June | Communicate the in-person meeting outcomes and proposed governance structure to staff and community members. | 
+|July | Present motions to approve the proposed structure of the governing board, executive leadership, and transfer of assets (e.g. finances, cross-organization subcommittees). Enumerate required bylaws.|
+|August | Present and approve an official motion to combine Software Carpentry and Data Carpentry into a unified organization. Approve nominations for the appointed seats to the board.|
+|September | Approve a unified budget. Approve the transfer of assets. |
+|October | Call for candidates for elected members of the board. Approve motions to dissolve the current governing boards and transfer assets as of January 1. |
```

Comment:
We wanted to pick a date, and given the timeline, it was looking like end of Dec / beginning of Jan would work, so Jan 1 seemed clean and tidy.